### PR TITLE
INPUT: Implement custom input for phone form

### PR DESCRIPTION
**Pull request by eunjuhuss**
#### Description:
Replace new customInput for Phone Form

----
before and after (with error message)@desktop 1200px
Screenshot 2020-08-12 at 13 42 08
before and after (with error message)@mobile 320px
Screenshot 2020-08-12 at 13 41 21

----

#### For reviewer:
- [x] Read the above description
- [x] Reviewed the code changes
- [x] Navigate to the branch (pulled the latest version)
- [x] Run the code and been able to execute the expected function
- [ ] Check any styling on both desktop and mobile sizes

### CODE REVIEW

File: src/login/styles/_buttons.scss [hunk 1]
```diff
@@ -95,10 +95,6 @@ button.close:hover {
   transform: scale(1.2);
 }
 
-button.btn#mobile-button {
-  margin-top: 0;
-}
-
 //  {
```
Comment (njons) [inline src/login/styles/_buttons.scss:98]: please delete this code if not in use for general clean up

**Comment by eunjuhuss** [comment]
this PR only changes to customInput and that validations we already have.
of course we can look into validation and make sure everything is working properly.

File: src/login/styles/_Emails.scss [hunk 2]
```diff
@@ -10,15 +10,8 @@
   margin-top: 1rem;
 }
 
+#emailsview-form,
 #phonesview-form {
-  flex-direction: row;
-  //align-items: baseline;
```
[the remark this answers is not in the document]
Comment (eunjuhuss) [reply src/login/styles/_Emails.scss:15]: this has been already removed

File: src/login/styles/_buttons.scss (hunk 1 above)
Comment (eunjuhuss) [reply src/login/styles/_buttons.scss:98]: this has been already removed

**Comment by eunjuhuss** [comment]
- Fixed label props errors in console: props type any
- Phone number placeholder translate text changed to capita letter
- made long error messages so now text is same indentation
- I don't think it will required a "required mark" because it is a active action to add an additional email/phone number

- from Johan, we can add foreign mobile numbers in EduID so I changed back phones.input_help_text(Phone number starting with 0 or +)that we had before.
- matched validation and error text so now it is required between 10-20 digits (previous the input require 6-20 digits)

File: src/login/styles/_Emails.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (eunjuhuss) [reply src/login/styles/_Emails.scss:15]: this had been already removed

**Review by njons — COMMENTED** [review]
👍   

2 quick questions: 
- Why is the label proptype `.any`, has it been re-engineers to take on props of different types? 
- Am I seeing this wrong or did the `letter-spacing` find it way back into `lable.scss`?

**Comment by eunjuhuss** [comment]
Why is the label proptype .any, has it been re-engineers to take on props of different types?
- Am I seeing this wrong or did the letter-spacing find it way back into lable.scss?

**Comment by eunjuhuss** [comment]
Q. Why is the label proptype .any, has it been re-engineers to take on props of different types?
-  label is string but label also have a translation. so the label is also an object. correct me if I'm wrong.

Q. Am I seeing this wrong or did the letter-spacing find it way back into lable.scss? 
- ` letter-spacing` in `label.scs`s has been returned to 0.5px, because other label spacing will be effected. so I removed` letter-spacing` `.input-label-helptext-container`.

**Review by njons — APPROVED** [review]
👍 

Agreed to move forward with a less specific proptype for `labels` (both object and string) for efficiency.

**Pull request merged**